var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -0,0 +1,54 @@
+from spack import *
+
+class Zoltan(Package):
+    """The Zoltan library is a toolkit of parallel combinatorial algorithms for
+       parallel, unstructured, and/or adaptive scientific applications.  Zoltan's
+       largest component is a suite of dynamic load-balancing and paritioning
+       algorithms that increase applications' parallel performance by reducing
+       idle time.  Zoltan also has graph coloring and graph ordering algorithms,
+       which are useful in task schedulers and parallel preconditioners."""
+
+    homepage = "http://www.cs.sandia.gov/zoltan"
+    base_url = "http://www.cs.sandia.gov/~kddevin/Zoltan_Distributions"
+
+    version('3.83', '1ff1bc93f91e12f2c533ddb01f2c095f')
```

Comment:
The version numbering scheme used for Zoltan is certainly a bit odd, but it seems to adhere to the `x.yz` scheme in a way that's consistent with Spack's version expectations so there shouldn't be an issue.  This is certainly a good point though; I'll keep this in mind just in case the Zoltan developers ever decide to switch their version numbering scheme and we run into problems with Zoltan versions.